### DISCUSSION TITLE
Calendar 24h time picker is resetting

### DISCUSSION THREAD

**Discussion by Ruba008** [primefaces/community#1390] [PrimeVue]
Hey guys. Unfortunelly, I've problems with the primevue calendar component, specifically the time picker. When I set a new date in my calendar, the time picker reset the hour to 00, and the minutes keeps the same. It's really bad for the user experience. 
How could I solve this problem? Is there another way?

Thanks for attention.

**Comment by tugcekucukoglu** [comment]
Hey. Could you send a reproducer link?

Thanks,

**Reply by Y0niix** [reply]
Hey. You can reproduce it on the official doc, with the 24hformat : https://primevue.org/calendar/#time
Regards.

**Reply by m-meier** [reply]
This issue should be fixed with primevue 3.52, however it still persists in v4-b.2.
